package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@bahmutov/parse-github-repo-url",
   "description": "Parse a GitHub URL for user/project@version",
-  "version": "0.0.0-semantic-release",
```

Comment:
please put the semantic release back - this module is published by TravisCI automatically including determining the version